clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -0,0 +1,260 @@
+from grafanalib import core as g
+
+DECREASING_ORDER_TOOLTIP = g.Tooltip(sort=g.SORT_DESC)
+DEFAULT_PANEL_HEIGHT = g.Pixels(300)
+
+
+def simple_graph(title, exprs, yAxes=None):
+    if not isinstance(exprs, (list, tuple)):
+        exprs = [exprs]
+    return g.Graph(
+        title=title,
+        dataSource="$source",
+        # One graph per row.
+        span=g.TOTAL_SPAN,
+        targets=[g.Target(expr=expr) for expr in exprs],
+        yAxes=yAxes or g.YAxes(),
+        tooltip=DECREASING_ORDER_TOOLTIP,
+    )
+
+
+clusterloader_panels = [
```

Comment:
I think those variables should be treated as consts. Please use all caps (https://github.com/google/styleguide/blob/gh-pages/pyguide.md#316-naming)

Reply:
Done